cmd/dtab.go

```diff
@@ -123,18 +122,20 @@ var (
 				if err != nil {
 					return err
 				}
-				versioned, err := ctl.Get(name)
-				if err != nil {
-					return err
+				switch len(args) {
+				case 2:
+					_, err = ctl.Update(name, dtabstr, namer.Version(""))
+				case 3:
+					_, err = ctl.Update(name, dtabstr, namer.Version(args[2]))
+				default:
+					return errors.New("update requires a name and file path")
 				}
-				_, err = ctl.Update(name, dtabstr, versioned.Version)
 				if err != nil {
 					return err
 				}
 				fmt.Printf("Updated %s\n", name)
 				return nil
-
-			default:
+			} else {
```

Comment:
That's way better, thanks.  I think the part of my brain that knows about mutable variables has completely atrophied.